### ISSUE TITLE
Store the packetID instead of the entire packet for forwarded packets

### ISSUE THREAD

**Issue by colin-axner** [cosmos/ibc-go#6636]
## Summary

Following https://github.com/cosmos/ibc-go/issues/6635, transfer could store only the `packetID` for the packet received which generated a forwarded packet. Currently we store the entire packet, when only the `packetID` information is needed

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged/assigned

**Comment by NisTun** [comment]
Can I handle this issue @colin-axner ?

**Comment by colin-axner** [comment]
Hi, @NisTun. Thanks for asking and thanks for your interest. Unfortunately you cannot, the issue is on hold and requires changes to core IBC before it can be worked on. Once that is done, I think the team would like to handle this issue ourselves

**Comment by colin-axner** [comment]
This issue is on hold until https://github.com/informalsystems/hermes/issues/4078 is addressed. I have moved the issue out of the v9 milestone and into v10, but the exact release it can be addressed in is dependent on the relayers

**Comment by gjermundgaraba** [comment]
Since forwarding has been removed, this is no longer relevant

**Comment by gjermundgaraba** [comment]
This issue was mistakenly reopened during some integration work, so closing again now.
